src/asciidoc/docs/develop/app-dev-process.adoc

```diff
@@ -0,0 +1,204 @@
+
+. Define the instructions for building your code and deploying your app on the Koji platform.
++
+Learn more: <<projects#_develop, develop key>>, <<projects#_deploy, deploy key>>
+
+. If the app has customization options for creators, define the default values for those options and any placeholder values for new, customized versions.
++
+Learn more: <<projects#_remixData, remixData key>>, <<projects#_@@initialTransform, @@initialTransform key>>
+
+. Configure the entitlements for the Koji platform features in your app.
++
+Learn more: <<entitlements#>>
+
+=== Add @withkoji/core to the frontend
+
+Follow these steps to use the @withkoji/core package in the frontend of your app.
+
+[TIP]
+====
+Before you start developing your code, check for outdated dependencies in the forked app.
+
+* If the forked app uses deprecated Koji packages, migrate your project to use the @withkoji/core package.
++
+ Learn more: <<migrate-koji-core#_do_i_need_to_migrate, Do I need to migrate?>>
+
+* Check for other outdated dependencies in the forked app, and update them if necessary.
++
+ Learn more: <<work-locally#_installing_the_dependencies, Installing dependencies>>
+====
+
+. In your frontend directory, install or update the @withkoji/core package.
++
+[source, Bash]
+npm install @withkoji/core
+
+. In your frontend code, import the package.
++
+[source,Javascript]
+import Koji from '@withkoji/core';
+
+. Use the `Koji.config` function to initialize the package with your app's configuration data before any data in the application is rendered.
+For example:
++
+[source,Javascript]
+----
+// Initialize
+Koji.config(require('././koji.json'));
+
+// render
+render();
+----
+
+. Use the `Koji.ready` function to indicate that the application is ready to start receiving events.
+
+. Implement the desired core functionality.
+For example, you can provide customization options so creators can add their own content and branding to the app before adding it to their profiles.
++
+Learn more: <<core-frontend-overview#>>, <<capture-user-input#>>, <<thumbnail-image#>>
+
+=== Add @withkoji/core to the backend
+
+If your app has a backend service, follow these steps to use the @withkoji/core package in it.
+
+. In your backend directory, install or update the @withkoji/core package.
++
+[source, Bash]
+npm install @withkoji/core
+
+. In your backend code, import the package and your app's configuration data.
++
+[source,Javascript]
+----
+import { KojiBackend } from '@withkoji/core';
+
+import kojiConfig from '../../koji.json';
+----
+
++
+TIP: To initialize the core package in your backend, the `koji.json` file must be bundled with the backend service during deployment of your app.
+Many frameworks won't bundle this file by default because it is not in the same folder as the backend code.
+To solve this problem, you can use a tool such as https://www.npmjs.com/package/babel-plugin-inline-json-import[babel-plugin-inline-json-import].
```

Comment:
Maybe generalize the type of tool required, instead of giving a specific example. If the developer of this tool pulls it out of the market or if it stops working, our readers wouldn't know how to find another solution.

Reply:
Added "to ensure the configuration data is available to your backend service". Again, something to note for future versions.